internal/bminventory/inventory.go

```diff
@@ -5413,11 +5413,13 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 	}
 
 	if err = b.customizeHost(host); err != nil {
+		// TODO Need event for infra-env instead of cluster
```

Comment:
please add ticket number to those comments `MGMT-7322` will be easier to find them later.